Getting error

After updating to the latest version, I am getting the following error:

> Cannot find module '@angular-devkit/core'

Not sure why, because I am only using `@nx-extend/pulumi`. Any idea of why this may have started happening?

--- a/packages/pulumi/executors.json
+++ b/packages/pulumi/executors.json
@@ -9,6 +9,11 @@
       "implementation": "./src/executors/preview/preview.impl",
       "schema": "./src/executors/preview/schema.json",
       "description": "preview executor"
+    },
+    "refresh": {
+      "implementation": "./src/executors/refresh/refresh.impl",
+      "schema": "./src/executors/refresh/schema.json",
+      "description": "refresh executor"
     }
   },
   "builders": {

